prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java

```diff
@@ -5,13 +5,19 @@
 import com.prefect.office.record.management.data.dao.prefect.offense.OffenseDao;
 import com.prefect.office.record.management.data.dao.prefect.offense.impl.OffenseDaoImpl;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class OffenseFacadeImpl implements OffenseFacade {
 
     OffenseDao offenseDao = new OffenseDaoImpl();
 
     public OffenseFacadeImpl(OffenseDao offenseDao){
```

Comment:
why is this necessary?

Reply:
sir import java.util.List; is necessary po because of the code 

public List<Offense> getAllOffenses() {
        return offenseDao.getAllOffenses();
    }